package.json

```diff
@@ -75,7 +79,7 @@
     "switchery-latest": "^0.8.2",
     "three": "~0.97.0",
     "universal-ga": "^1.2.0",
-    "vue": "2.6.12"
+    "vue": "^2.6.14"
```

Comment:
please keep the pinned dependencies as they are not related to this PR

Reply:
if you install it from master, 2.6.12 vue gets installed together with 2.6.14 vue-template-compiler. This just fixed both to the same version, so nothing breaks.